packages/components/src/Select/Select.tsx

```diff
@@ -0,0 +1,261 @@
+import React, { useId, useState } from "react"
+import classnames from "classnames"
+import ReactSelect, {
+  components,
+  Props as ReactSelectProps,
+  NoticeProps,
+} from "react-select"
+import Async, { AsyncProps as ReactAsyncSelectProps } from "react-select/async"
+import { FieldMessage } from "~components/FieldMessage"
+import { ChevronDownIcon, ChevronUpIcon, ClearIcon } from "~components/Icon"
+import { Label } from "~components/Label"
+import { Tag } from "~components/Tag"
+import styles from "./Select.module.scss"
+
+export type { OnChangeValue as SelectValueType } from "react-select"
```

Comment:
Strange thing to export.
It's not used here though, so export it through `index.ts` instead (or make a `types` file if you want).